west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: d7af6f371034f31b9440b27c694b0be3c87491d3
+      revision: 04de43b1a119665532b6b2a0c6bbcbefd47eb7d6
```

Comment:
For the mergeable version, this needs to be combined with the previous git commit. Otherwise git-bisect will be broken as the first commit cannot be compiled with old zephyr so this has to be a single git commit to change the API and modify west.yml.

Reply:
thanks